This patch takes care of setting up the configure script in order to install
all the needed m4 files that store the macros that are then used by packages
that use libsc in order to setup correct config options
Author: @rubendibattista

--- Makefile.old	2019-03-05 18:37:50.912827712 +0100
+++ Makefile.am	2019-03-05 18:54:16.272505551 +0100
@@ -36,7 +36,9 @@
         config/sc_c_version.m4 config/sc_cuda.m4 config/sc_include.m4 \
         config/sc_lapack.m4 config/sc_lint.m4 config/sc_mpi.m4 \
         config/ax_prefix_config_h.m4 config/ax_split_version.m4 \
-        config/sc_package.m4 config/sc_trilinos.m4
+        config/sc_package.m4 config/sc_trilinos.m4 config/sc_pthread.m4 \
+		config/sc_openmp.m4 config/sc_memalign.m4
+
 
 # install example .ini files in a dedicated directory
 scinidir = $(datadir)/ini
